projects/demo/src/modules/interfaces/front-end-example.ts

```diff
@@ -1,5 +1,11 @@
+export const EXAMPLE_MAIN_FILE_NAME = {
```

Comment:
What do you think about reuse this code in `addon-doc`?

https://github.com/TinkoffCreditSystems/taiga-ui/blob/90195ec9b83493687f1db146132faf85160ee448/projects/addon-doc/src/interfaces/page.ts

https://github.com/TinkoffCreditSystems/taiga-ui/blob/90195ec9b83493687f1db146132faf85160ee448/projects/demo/src/modules/interfaces/front-end-example.ts

Reply:
i can reuse it later when your code will be merged to `main` branch